notes/meetings/2021-12/december-15.md

```diff
@@ -0,0 +1,66 @@
+# December 15, 2021 - Meeting Notes
+
+**Faciliators:**
+
+| Name | Abbreviation |
+| - | - | - |
+| Dominik Wilkowski | DW |
+| Esther Semo | ES |
+| Matt Sawks | MS |
+| Elise Chant | EC |
+
+## Resources
+
+- [Survey Summary Results](assets/survey-results.pdf)
+- [Survey Full Results](assets/survey-results-full.csv)
+
+## Prior
+
+* Maintainers have read proposed RFCs
+
+## Survey findings
+
+* MS to present
+* Results at assets/survey-results.pdf and assets/survey-results-full.csv
+
+## Prioritisation of actions related to survey findings
+
+* MS to facilitate
+* Actions to be created in GitHub
+
+## Creating a skills register
+
+* DW to facilitate
+  * Creation of new Slack channels
+    * Technical group
```

Comment:
I think we should create these ahead of the meeting, can we agree to naming conventions? 

how about:

#maintainers-technical
#maintainers-design
#maintainers-a11y
#maintainers-content